packages/calling/src/CallingClient/calling/CallerId/index.test.ts

```diff
@@ -132,6 +132,29 @@ describe('CallerId tests', () => {
     expect(callerId['callerInfo'].num).toStrictEqual('5008');
   });
 
+  it('When PA-ID, From header is present for guest user', async () => {
+    const dummyCallerId = {
+      'x-broadworks-remote-party-info':
+        'userId="nkjwuovmbo@64941297.int10.bcld.webex.com";userDn="tel:+12142865888;ext=5888;country-code=1";externalId=69fde5ad-fb8b-4a1b-9998-b0999e95719b',
+      'p-asserted-identity':
+        '"Bob Marley  -  SdkGuestCall (Guest)" <sip:5888@10.155.4.7;user=phone>',
+      from: '"Alice -  SdkGuestCall (Guest)" <sip:5889@64941297.int10.bcld.webex.com>;tag=1932136170-1654008881246',
+    };
+
+    const output = callerId.fetchCallerDetails(dummyCallerId);
+
+    /* PAI should be preferred */
+    expect(output.name).toStrictEqual('Bob Marley  -  SdkGuestCall (Guest)');
+    expect(output.num).toStrictEqual('5888');
+    await waitForMsecs(50);
+
+    /* Should be overwritten by x-broadworks */
+    expect(callerId['callerInfo'].avatarSrc).toStrictEqual(dummyAvatar);
+    expect(callerId['callerInfo'].id).toStrictEqual(dummyScimResponse.Resources[0].id);
+    expect(callerId['callerInfo'].name).toStrictEqual('Cathy');
+    expect(callerId['callerInfo'].num).toStrictEqual('5008');
```

Comment:
Do we really need this? Isn't the previous UT already checking this?

Reply:
Updated tests and test names